.github/workflows/ci.yml

```diff
@@ -375,7 +375,7 @@ jobs:
           - linux
           - macos
           #- windows # TODO: Fix "windows cannot load linux image on windows".
-    runs-on: ${{ (contains('ios macos', matrix.platform) && 'macos-13')
+    runs-on: ${{ (contains('ios macos', matrix.platform) && 'macos-15-intel')
```

Comment:
`macos-13` is deprecated so remove it. I've also tried `macos-latest` but it had some docker installation errors.